### PR TITLE
[14.0][MIG] base comment template - moved from account-invoice-reporting

### PR DESCRIPTION
**Pull request by petrus-v**
replace https://github.com/OCA/reporting-engine/pull/473 and https://github.com/OCA/account-invoice-reporting/pull/163

Please note that @alan196 already review previous PR on [account-invoice-reporting repo](https://github.com/OCA/account-invoice-reporting/pull/163) I've only change repo name reference since his review.

### CODE REVIEW

**Review by alan196 — APPROVED** [review]
:+1:

**Review by SimoneVagile — APPROVED** [review]

File: base_comment_template/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+Two positions are available for the comments:
+* above document lines
+* below document lines
+
+This module is the base module for following modules:
+
+* sale_comment_template
+* purchase_comment_template
+* invoice_comment_template
```
Comment (emagdalenaC2i) [inline base_comment_template/readme/DESCRIPTION.rst:12]:
```suggestion
* invoice_comment_template
* stock_picking_comment_template
```
See https://github.com/OCA/stock-logistics-reporting/tree/12.0/stock_picking_comment_template

File: base_comment_template/readme/CONTRIBUTORS.rst [hunk 2]
```diff
@@ -0,0 +1,14 @@
+* Xavier Jimenez <xavier.jimenez@qubiq.es>
+* Nicolas Bessi <nicolas.bessi@camptocamp.com>
+* Yannick Vaucher <yannick.vaucher@camptocamp.com>
+* Guewen Baconnier <guewen.baconnier@camptocamp.com>
+* Simone Rubino <simone.rubino@agilebg.com>
+* Simone Rubino <simone.rubino@agilebg.com>
```
Comment (emagdalenaC2i) [inline base_comment_template/readme/CONTRIBUTORS.rst]: This line is duplicated

**Review by emagdalenaC2i — COMMENTED** [review]
LGTM. Just a few minor changes
Please squash your commits

**Review by emagdalenaC2i — APPROVED** [review]

File: base_comment_template/readme/CONTRIBUTORS.rst (hunk 2 above)
Comment (petrus-v) [reply base_comment_template/readme/CONTRIBUTORS.rst]: I've removed the duplicated line and squash last commits

**Review by rousseldenis — APPROVED** [review]

**Comment by emagdalenaC2i** [comment]
@OCA/reporting-engine-maintainers Please, could you merge this PR?
There are other PRs that depends of this one

**Review by stevech091 — APPROVED** [review]
Functional test OK

**Review by alan196 — APPROVED** [review]

**Pull request merged**